src/governance.spec.ts

```diff
@@ -77,4 +84,62 @@ describe("GovernanceCanister.listKnownNeurons", () => {
 
     expect(res.map((n) => Number(n.id))).toEqual([100, 200, 300, 400]);
   });
+
+  it("creates new neuron successfully", async () => {
+    const clainNeuronResponse: ClaimOrRefreshNeuronFromAccountResponse = {
+      result: [{ NeuronId: { id: BigInt(0) } }],
+    };
+    const service = mock<GovernanceService>();
+    service.claim_or_refresh_neuron_from_account.mockResolvedValue(
+      clainNeuronResponse
+    );
+
+    const mockLedger = mock<LedgerCanister>();
+    mockLedger.transfer.mockImplementation(jest.fn());
+
+    const governance = GovernanceCanister.create({
+      certifiedServiceOverride: service,
+    });
+    await governance.stakeNeuron({
+      stake: ICP.fromString("1") as ICP,
+      principal: new AnonymousIdentity().getPrincipal(),
+      ledgerCanister: mockLedger,
+    });
+
+    expect(mockLedger.transfer).toBeCalled();
+    expect(service.claim_or_refresh_neuron_from_account).toBeCalled();
+  });
+
+  it("gets user neurons", async () => {
```

Comment:
Just for my understanding, this test is completely unrelated to this PR right? If yes, probably better to do introduce it separately.